packages/components/cascader/src/cascader.vue

```diff
@@ -252,7 +252,7 @@ const popperOptions: Partial<Options> = {
       fn: ({ state }) => {
         const { modifiersData, placement } = state as any
         if (['right', 'left', 'bottom', 'top'].includes(placement)) return
-        modifiersData.arrow.x = 35
+        modifiersData.arrow && (modifiersData.arrow.x = 35)
```

Comment:
I think it makes more sense to use an if statement here.
```suggestion
        if (modifiersData.arrow) {
          modifiersData.arrow.x = 35
        }
```